### PR TITLE
fix: Fix up email links

### PR DESCRIPTION
**Pull request by DanCorderSoftwire**
Make email links point to the right place

### CODE REVIEW

File: SeaPublicWebsite/Constants.cs [hunk 1]
```diff
@@ -2,5 +2,6 @@
 
 public static class Constants
 {
+    public const string FEEDBACK_URL = "https://sea-beta-dev.cloudapps.digital/feedback";
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite/Constants.cs:5]: This doesn't work on prod, but we're about to change feedback to be a separate site so then one link will make sense.

File: SeaPublicWebsite/appsettings.Production.json [hunk 2]
```diff
@@ -7,5 +7,9 @@
                 "Args": { "uri": "tls://34ae15fb-5760-47f5-a1b8-93568b5df4d3-ls.logit.io:17202" }
             }
         ]
+    },
+
+    "GovUkNotify": {
+        "BaseUrl": "https://sea-beta-production.cloudapps.digital/"
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite/appsettings.Production.json:13]: This doesn't have /energy-efficiency which seems correct, but the one below has

**Review by CodrinCotarlanSoftwire — CHANGES_REQUESTED** [review]
Fix discrepancy between prod and dev config base url

File: SeaPublicWebsite/appsettings.Production.json (hunk 2 above)
Comment (DanCorderSoftwire) [reply SeaPublicWebsite/appsettings.Production.json:13]: Thanks, that's what I get for rushing

**Review by CodrinCotarlanSoftwire — APPROVED** [review]

**Pull request merged**